svc/wallet/types.go

```diff
@@ -130,11 +130,8 @@ type (
 
 	// Staking struct
 	Staking struct {
-		AssetName   string  `json:"asset_name"`
-		APY         float64 `json:"apy"`
 		TotalStaked float64 `json:"total_staked"`
 		Staked      float64 `json:"staked"`
-		YourShare   float64 `json:"your_share"`
 	}
```

Comment:
the struct will be helpful when we add true staking (it's already in the roadmap), so add `omitempty` to optional fields